src/codegen/Common.ts

```diff
@@ -710,6 +710,102 @@ const common = {
     )
       .setTitle('Layout set')
       .setDescription('Settings regarding a specific layout-set'),
+  IFormatting: () =>
```

Comment:
If this is duplicated now, maybe the definition in the `Input` component should reference this instead?